SU-SWS/stanford_sites_helper
# SITES-000: Add user-facing report of file paths

# READY FOR REVIEW

# Summary
- Let users look up their file paths

# Needed By (Date)
- 10.8.18

# Criticality
- How critical is this PR on a 1-10 scale? 4/10
- It would be nice to have once we cut over 800 "people" sites and have to help users with broken links to files

# Steps to Test

1. Check out this branch and enable the module
2. Go to `admin/reports/status`
3. See "File Paths | Your public file path is sites/default/files. Your private file path is sites/default/files/private." (or whatever your file paths are)

# Affected Projects or Products
- ACSF

# Associated Issues and/or People
None

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
